docs/source/_static/custom.css

```diff
@@ -0,0 +1,9 @@
+/* 24px margin from readthedocs theme */
+div.nblast {
+    margin-bottom: 19px !important;  /* padding has already 5px */
```

Comment:
`.nblast` is used wherever there are code cells, e.g. in https://jupyter-notebook.readthedocs.org/en/latest/examples/Notebook/Running%20Code.html

I didn't want to create a separate reST directive as container for a "code cell" (i.e. the combination of an `nbinput` directive and 0..N `nboutput` directives). Currently, the `nbsphinx` directives for input/output cells can be used on the top level, see http://nbsphinx.readthedocs.org/en/latest/rst.html#sphinx-directives-for-jupyter-notebook-cells.
Since I don't have a reST container, I also don't know how to get the parts of a "code cell" into a CSS container like the `code_cell` class used by the Notebook/`nbviewer`. If you know a (reasonably simple) way to do that, please tell me!
As a work-around, I introduced the [:no-output:](https://github.com/spatialaudio/nbsphinx/blob/2181601ed469f309d649732719adcb354f3240ad/nbsphinx.py#L74) and [:more-to-come:](https://github.com/spatialaudio/nbsphinx/blob/2181601ed469f309d649732719adcb354f3240ad/nbsphinx.py#L102) flags for the `nbinput`/`nboutput` directives, which translate to the `nblast` CSS class. This class is used in the last group of a "code cell", i.e. in an input cell if no output is available or in an output cell if it's the last of a group of outputs.

Since (currently) there is no CSS container class for a "code cell" in `nbsphinx`, I cannot use `:last-child`. I also wouldn't have to if there were, because then I could simply use the `margin` settings of the container.

But probably I can do this with some other CSS magic?